style.css

```diff
@@ -48,4 +49,151 @@ body {
 .text {
   color: #c8c8d4;
   font-size: 20px;
+  font-weight: 400;
+}
+
+#section-2 {
+  display: flex;
+  justify-content: center;
+}
+
```

Comment:
- [ ] Check this out, the user will not be able to see the text behind this light image if I can call it so 😄, please adjust it a bit to make it match the original design
- [ ] Your see project button has not the proper width, please fix it.

<img width="314" alt="see" src="https://user-images.githubusercontent.com/79058364/159077888-feb7818a-7f22-4406-8fc1-390199163fb4.png">

<img width="308" alt="see1" src="https://user-images.githubusercontent.com/79058364/159077944-a668b9be-8e68-41c3-8466-6b409772b83a.png">

- it should be like this instead,
<img width="228" alt="zero" src="https://user-images.githubusercontent.com/79058364/159078271-f81359e5-bf4b-49b3-91ef-5679404ecd4f.png">